core/tools/nginx/nginx.conf

```diff
@@ -187,6 +187,14 @@ http {
             try_files $uri $uri/ /index.html;
         }
 
+        location /userdata {
+            root /usr/blueos;
+            autoindex on;
+            # do we want these next two lines?
+            expires -1;
+            add_header Cache-Control no-store;
```

Comment:
It makes sense to disable cache, makes development easier... And I don't think it would impact that much on loading performance.